modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/am/integration/tests/logging/CorrelationLoggingTest.java

```diff
@@ -310,7 +313,7 @@ public void testPersistedCorrelationConfigs() throws Exception {
         InvokeTestAPI();
         while ((logLine = bufferedReader.readLine()) != null) {
             log.info("***p5 logLine: " + logLine);
-            assertFalse(isHTTPLogLine(logLine) || isMethodCallsLogLine(logLine));
+            assertFalse(isHTTPLogLine(logLine) && isMethodCallsLogLine(logLine));
```

Comment:
This will pass always. 
Because below statement is false always. The log is not of both type always.
`isHTTPLogLine(logLine) && isMethodCallsLogLine(logLine)`

Reply:
Fixed in https://github.com/wso2/product-apim/pull/13084/commits/ad7212fdc81f282d1450e4bbc2eb6d51968424c8